core/src/test/java/greencity/controller/PlaceControllerTest.java

```diff
@@ -606,4 +580,126 @@ void getAllPlaces() throws Exception {
         verify(placeService, times(1))
             .findAll(pageable, principal);
     }
+
+    @Test
+    void updateStatusWithApprovedStatusAPPROVED() throws Exception {
+        UpdatePlaceStatusWithUserEmailDto dto = new UpdatePlaceStatusWithUserEmailDto();
+        dto.setPlaceName("testPlace");
```

Comment:
mb, move creation of this dto to model utils, because it is used in 4 tests and the only difference is newStatus